src/test/java/runner/TestRunner.java

```diff
@@ -1,11 +1,36 @@
 package runner;
 
+import desktop.pages.LoginPage;
+import driver.SingletonDriver;
+import org.junit.jupiter.api.Test;
 import org.junit.platform.runner.JUnitPlatform;
 import org.junit.platform.suite.api.SelectPackages;
 import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 
 @RunWith(JUnitPlatform.class)
 @SelectPackages("put your packages here")
+class TestRunner {
+
+    @Test
+    void loginPageTest() {
+
+        String namePlaceholder = "Name";
+        String emailPlaceholder = "Your email address";
+        String passwordPlaceholder = "Create a password";
+
+        WebDriver driver = SingletonDriver.getWebDriverInstance();
+        driver = SingletonDriver.openPage(driver, "https://www.bookdepository.com/account/login/to/account");
```

Comment:
you can try to open main page first and then click on this sign in link that would be in MainHeader fragment

Reply:
done.